build.gradle

```diff
@@ -2,23 +2,23 @@
 buildscript {
     repositories {
         google()
-        jcenter()
         mavenCentral()
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'com.android.tools.build:gradle:8.0.0'
         classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.8.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
         classpath 'org.codehaus.groovy:groovy-all:2.4.15'
     }
 }
 
 allprojects {
-  repositories {
-    google()
-    jcenter()
-    maven { url "https://jitpack.io" }
-    maven { url "https://maven.google.com" }
-  }
+    repositories {
```

Comment:
Replaced [deprecated](https://docs.gradle.org/7.5/userguide/upgrading_version_6.html#jcenter_deprecation) jcenter() convenience method with gradlePluginPortal() as JCenter was [shutdown](https://blog.gradle.org/jcenter-shutdown).

Gradle Plugin Portal was added as the JCenter dependency [CircleProgressBar 1.1.1](https://mvnrepository.com/artifact/com.dinuscxj/circleprogressbar/1.1.1) was not available on Maven Central.

Refer to this [stack overflow issue](https://stackoverflow.com/questions/66651640/jcenter-deprecation-impact-on-gradle-and-android) and this [github issue](https://github.com/gradle/gradle/issues/4020).